flink-end-to-end-tests/test-scripts/common_kubernetes.sh

```diff
@@ -23,7 +23,7 @@ source "$(dirname "$0")"/common_docker.sh
 CONTAINER_SCRIPTS=${END_TO_END_DIR}/test-scripts/container-scripts
 RETRY_COUNT=3
 RETRY_BACKOFF_TIME=5
-RESULT_HASH="e682ec6622b5e83f2eb614617d5ab2cf"
+RESULT_HASH="d41d8cd98f00b204e9800998ecf8427e"
```

Comment:
Where is this variable used?
Is it in `flink-end-to-end-tests/test-scripts/test_kubernetes_embedded_job.sh`

If so, can you move the variable to this script?

Reply:
This variable is used in tests on k8s e2e, but it is basically covered by the test script.

The hash value needs to be changed because the test jobs have basically changed from the wordcount example job of the _dataset_ API to the _datastream_ API. The output results have changed, for example, from a single file to a directory, and the output format has changed from _'word 1'_ to _'(word , 1)'_.

I didn't remove it because I didn't want this PR to look too weird and require a variable to be suddenly deleted.
But thanks for your suggestion, I will recheck the usage scope of this variable and try to delete this variable in _common_k8s_ script.